KsApi/models/templates/GraphMutationPostCommentResponseEnvelopeTemplate.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+extension GraphMutationPostCommentResponseEnvelope {
+  internal static let template =
+    GraphMutationPostCommentResponseEnvelope(createComment: .init(comment: .init(
+      body: "Hello World",
+      id: "Q29tbWVudC0zMjY2MjU0MQ=="
+    )))
+}
```

Comment:
Wrote this to help with mocking the request, in a test, but I don't think I can do any assertions until we have some outputs in the view model to observe?

Reply:
You could output a templatized `Comment`.

Reply:
Added an output thats emitting the template on viewDidLoad and wrote a small test for it.